HACKING.rst

```diff
@@ -1,3 +1,14 @@
+Running YAPF on itself
+----------------------
+
+The run YAPF on all of YAPF::
```

Comment:
As a non-native English speaker, I don't understand what this means. Without 'The', however, it makes sense to me.